app/src/main/java/com/example/tasky/login/components/ButtonWithLoader.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.tasky.login.components
```

Comment:
I like to separate the packages using Feature-Layered (check the video about the Right Package Structure in the mentorship video portal)

In this case, a feature could be "Authentication", and inside it separate it into presentation, data and domain

In this case, this could be inside:
Authentication-presentation-login-components